src/types/template.paths.ts

```diff
@@ -6,6 +6,7 @@ export enum Templates {
     SERVICE_OFFLINE_MID_JOURNEY = "service-offline-mid-journey",
     SIC = "tasks/sic",
     START = "start",
+    STATEMENT_OF_CAPITAL = "statement-of-capital",
```

Comment:
could you move the template to the tasks subfolder - views/tasks/. i think thats where we'll be storing the templates for the tasks